rs/nns/governance/tests/governance.rs

```diff
@@ -293,6 +293,14 @@ fn test_single_neuron_proposal_new() {
                             ),
                         ),
                         NeuronChange::RecentBallotsNextEntryIndex(OptionChange::BothSome(U32Change(0, 1,),),),
+                        NeuronChange::DecidingVotingPower(
+                            OptionChange::BothSome(
+                                U64Change(
+                                    1,
+                                    0,
+                                ),
+                            ),
+                        ),
```

Comment:
Frankly, I do not understand why this happens. These `assert_changes!` have always seemed messed up to me. They can explode even when there is nothing wrong (and they often do). As a result, I now have a "boy who cried wolf" attitude towards these. If you believe these indicate genuine bugs, please, lmk.

Reply:
I'm a bit confused as to why this PR changes the test spec. Let's sync on this tomorrow?

Reply:
> I'm a bit confused as to why this PR changes the test spec.

Me too. I thought this was clear from what I said originally?

> Let's sync on this tomorrow?

👍 although, since we are both confused, I'm not sure what we'll be able to discuss :S

Reply:
Discussed over zoom, this seems expected that the fields would change.